src/components/qa/exercises.cjsx

```diff
@@ -54,11 +54,9 @@ QAExercises = React.createClass
 
   renderExerciseContent: (exercises) ->
     exercises = _.map exercises, (exercise) =>
-      hideAnswers = @state.isShowing2StepPreview and
-        ExerciseStore.hasQuestionWithFormat('free-response', {exercise})
       <ExerciseCard key={exercise.id}
         exercise={exercise}
-        hideAnswers={hideAnswers}
+        show2StepPreview={@state.isShowing2StepPreview}
         ignoredTypes={@state.ignored} />
 
     selections = _.map ExerciseStore.getPageExerciseTypes(@state.pageId), (pt) =>
```

Comment:
An idea.  How about if we have the exercise preview set the question's `format` as either a class or maybe  `data-format` attributes.  We could then target only questions that have `free-response` via css and show the two step preview on only them.

Thoughts?